sdk/python/foundation-models/healthcare-ai/medimageparse/mip-deploy.ipynb

```diff
@@ -136,7 +136,7 @@
     "    name=deployment_name,\n",
     "    endpoint_name=endpoint_name,\n",
     "    model=model,\n",
-    "    instance_type=\"Standard_NC6s_v3\",\n",
+    "    instance_type=\"STANDARD_NC4AS_T4_V3\",\n",
```

Comment:
Use H100 or A100.  T4s are too small for MIP.  Standard_NC40ads_H100_v5 is good.